source/CMakeLists.txt

```diff
@@ -188,89 +188,61 @@ add_custom_target(spirv-tools-build-version
    DEPENDS ${SPIRV_TOOLS_BUILD_VERSION_INC})
 set_property(TARGET spirv-tools-build-version PROPERTY FOLDER "SPIRV-Tools build")
 
-add_subdirectory(comp)
-add_subdirectory(opt)
-add_subdirectory(link)
+add_library(${SPIRV_TOOLS} "")
 
-set(SPIRV_SOURCES
-  ${spirv-tools_SOURCE_DIR}/include/spirv-tools/libspirv.h
+include(GenerateExportHeader)
```

Comment:
Why not make your own export header, it is simple to do and makes the code more clear imho.